components/brave_wallet/renderer/test/BUILD.gn

```diff
@@ -8,7 +8,10 @@ import("//testing/test.gni")
 
 source_set("brave_wallet_response_unit_tests") {
```

Comment:
please help rename this to a general unit_tests target.

Reply:
Fixed in d8c9c09e62fadd786f09bfdf932a2573f3dd5358